src/python_framework_v02/troute/network/reach.pxd

```diff
@@ -0,0 +1,25 @@
+cimport numpy as np
```

Comment:
I think we can drop the .pxd; and __init__.py

I ran a test with an empty troute folder in the python site-packages and no .pxd (nor __init__.py for that matter) and was able to install without trouble.

Reply:
What do you mean by "drop the .pxd"?